components/atoms/Avatar/avatar-hover-card.tsx

```diff
@@ -41,7 +41,7 @@ export const Avatar = ({ contributor, size = "large" }: AvatarProps): JSX.Elemen
   return (
     <Image
       alt={contributor}
-      className="border rounded-full"
+      className="border rounded-full not-prose"
```

Comment:
Adding the [not-prose](https://github.com/tailwindlabs/tailwindcss-typography#undoing-typography-styles) Tailwind CSS class was necessary to avoid the `prose` Tailwind class from adding padding to images.